### DISCUSSION TITLE
Frontend Update Idea: Mention the programming language next to the framework in the rankings

### DISCUSSION THREAD

**Discussion by bm1216** [TechEmpower/FrameworkBenchmarks#6787] [Ideas]
Hello,

I wanted to mention something that might be an improvement on the front-end side of things. Is it possible for the leaderboard to have a column (or something else) that can showcase the programming language next to the framework? I realize that they are marked as different colours based on the programming language, but the reference for the colours are in the filter which means you have to scroll up to consult it. Some of the colours are also similar, causing confusion. 

Thanks,
Barun

**Comment by joanhey** [comment]
Hi Barum,
Already exist a column for the language

![Captura de pantalla_área-de-selección_20211004222848](https://user-images.githubusercontent.com/249085/135920253-34e85b1b-e3a6-472d-b842-ecc22a011bd7.png)

Thanks
